packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -281,6 +283,13 @@ export class OdspDocumentService implements IDocumentService {
                 connection.on("op", (documentId, ops: ISequencedDocumentMessage[]) => {
                     this.opsReceived(ops);
                 });
+                // On disconnect with 401/403 error code, we can just clear the joinSession cache as we will again
+                // get the auth error on reconnecting and face latency.
+                connection.on("disconnect", (error: OdspError) => {
+                    if (error.errorType === DriverErrorType.authorizationError) {
```

Comment:
Do we guarantee that it's an object (OdspError)? If not, consider having error?.errorType check instead

Reply:
It seems it throws OdspError but it is not typed in the event. So replacing it with any and testing for object/null/errorType now.